relayer-cli/src/commands/config/auto.rs

```diff
@@ -0,0 +1,200 @@
+use crate::chain_registry::get_configs;
+use abscissa_core::{
+    clap::Parser,
+    {Command, Runnable},
+};
+
+use crate::conclude::Output;
+
+use ibc_relayer::{
+    config::{store, ChainConfig, Config},
+    keyring::{KeyRing, Store::Test},
+};
+
+use std::path::PathBuf;
+use tokio::runtime::Builder;
+use tracing::{info, warn};
+
+fn find_key(chainconfig: &ChainConfig) -> Option<String> {
+    if let Ok(keyring) = KeyRing::new(Test, &chainconfig.account_prefix, &chainconfig.id) {
+        if let Ok(keys) = keyring.keys() {
+            if let Some((name, _)) = keys.first() {
+                Some(name.to_string())
+            } else {
+                None
+            }
+        } else {
+            None
+        }
+    } else {
+        None
+    }
+}
+
+/// The data structure that represents the arguments when invoking the `config auto` CLI command.
+///
+/// The command has two required arguments and an optional one which is used to manually specify the keys to use in the config file:
+///
+/// `config auto [OPTIONS] --path <PATH> --chain <CHAIN_NAME_1 CHAIN_NAME_2...>`
```

Comment:
Do we really need the chain name here since there are embedded in the path JSON file?

Reply:
`--path` is just the path where the config will be stored. 

I chose not to overwrite `~/.hermes/config.toml` by default.